static/js/init.js

```diff
@@ -76,12 +76,12 @@ $(document).ready(function() {
     if ( $pageControls.length > 0 ) {
         var scrollPos,
             controlsHeight,
-            controlsTop = $pageControls.offset().top + 100;
+            controlsTop = $pageControls.offset().top - 41;
```

Comment:
Can we document what this constant represents and give it a name? I would also like to see this logic moved into its own module, either in this patch or a separate one.